xline/src/storage/kv_store.rs

```diff
@@ -922,8 +928,9 @@ mod test {
         );
         let db = DB::open(&StorageConfig::Memory)?;
         let store = init_store(db).await?;
-        let (_ignore, ops) = store.after_sync(&txn_req).await?;
+        let (sync_res, ops) = store.after_sync(&txn_req).await?;
         store.db.flush_ops(ops)?;
```

Comment:
If these two operations always happend togethor, put them in one function to force the "pair"?

Reply:
they always happened together in this module's tests, but not else where, I added a function to reduce duplication